lib/Doctrine/Search/Indexer.php

```diff
@@ -67,7 +67,9 @@ public function indexDirectory($dir)
         $coll = Doctrine_Collection::create('Doctrine_File');
 
         foreach ($files as $file) {
-            $coll[]->url = $file;
```

Comment:
This array access was frowned upon (I forget by what, perhaps psalm/phpstan).

The syntax below should be the same.